BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -249,19 +249,19 @@ An earlier version of this document was published as RFC 7525 when the industry
 
 # Introduction
 
-Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS) are widely used to protect data exchanged over application protocols such as HTTP {{HTTP1.1}} {{HTTP2}}, SMTP {{?RFC5321}}, IMAP {{?RFC9051}}, POP {{?STD53}}, SIP {{?RFC3261}}, and XMPP {{?RFC6120}}.  Over the years leading to 2015, the industry has witnessed serious attacks on TLS and DTLS, including attacks on the most commonly used cipher suites and their modes of operation.  For instance, both the AES-CBC {{?RFC3602}} and RC4 {{!RFC7465}} encryption algorithms, which together were once the most widely deployed ciphers, have been attacked in the context of TLS.  A companion document {{?RFC7457}} provides detailed information about these attacks and will help the reader understand the rationale behind the recommendations provided here. That document has not been updated in concert with this one; instead, newer attacks are described in this document, as are mitigations for those attacks.
+Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS) are widely used to protect data exchanged over application protocols such as HTTP {{HTTP1.1}} {{HTTP2}}, SMTP {{?RFC5321}}, IMAP {{?RFC9051}}, POP {{?STD53}}, SIP {{?RFC3261}}, and XMPP {{?RFC6120}}.  Over the years leading to 2015, the industry had witnessed serious attacks on TLS and DTLS, including attacks on the most commonly used cipher suites and their modes of operation.  For instance, both the AES-CBC {{?RFC3602}} and RC4 {{!RFC7465}} encryption algorithms, which together were once the most widely deployed ciphers, were attacked in the context of TLS.  Detailed information about the attacks known prior to 2015 is provided in a companion document ({{?RFC7457}}) to the previous version of this specification, which will help the reader understand the rationale behind the recommendations provided here. That document has not been updated in concert with this one; instead, newer attacks are described in this document, as are mitigations for those attacks.
```

Comment:
It's weird to talk about things that stopped in 2015 using a verb tense for events that occurred in the past and are still occurring.
Also, saying "A companion document" (to me) implies that it is a companion to this document, which is not exactly true -- it was a companion to 7525, but this document plays the role of both 7525 and the companion document for the changes since 2015.